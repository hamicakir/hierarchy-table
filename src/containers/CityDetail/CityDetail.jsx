import React from 'react';

const CityDetail = () => {

  return (
    <div>
      City Detail
    </div>
  )
};

export default CityDetail;
