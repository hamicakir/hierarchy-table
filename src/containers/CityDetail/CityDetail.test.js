import React from "react";
import ReactDOM from "react-dom";
import renderer from 'react-test-renderer';

import CityDetail from "./CityDetail";

describe("City Detail", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<CityDetail />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders correctly", () => {
    const tree = renderer.create(<CityDetail />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
