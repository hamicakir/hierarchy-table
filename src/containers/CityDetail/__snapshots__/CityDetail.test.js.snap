// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`City Detail renders correctly 1`] = `
<div>
  City Detail
</div>
`;
